src/java.base/windows/native/libjli/java_md.c

```diff
@@ -357,10 +357,10 @@ LoadJavaVM(const char *jvmpath, InvocationFunctions *ifn)
 
     /*
      * The Microsoft C Runtime Library needs to be loaded first. A copy is
-     * assumed to be present in the JDK installation root directory. If it
-     * is not found there (or the JDK installation root fails to resolve),
-     * skip the explicit load and let nature take its course, which is likely
-     * to be a failure to execute.
+     * assumed to be present within the JDK. If it is not found there
```

Comment:
I decided not to explicitly mention the "bin" directory here and let that be explained in the implementation of the `LoadMSVCRT()` function itself. If you think we should include the "bin" part here too, then let me know.

Reply:
That's okay, main thing is that the comment that said it was in the root directory has been fixed.